services/slack/export.go

```diff
@@ -280,8 +281,18 @@ func (t *Transformer) ExportDirectChannels(channels []*IntermediateChannel, writ
 }
 
 func (t *Transformer) ExportUsers(writer io.Writer) error {
-	for _, user := range t.Intermediate.UsersById {
-		line := GetImportLineFromUser(user, t.TeamName)
+	users := t.Intermediate.UsersById
+	userIds := make([]string, len(users))
+	i := 0
+	for k := range users {
+		userIds[i] = k
+		i++
+	}
+
+	sort.Strings(userIds)
+
+	for _, userId := range userIds {
+		line := GetImportLineFromUser(users[userId], t.TeamName)
```

Comment:
Instead of adjusting this source to be teastable, the test should be able to assert the entries without a guaranteed order

Reply:
Actually, keeping this as-is, to be able to make it so the output does not change across runs of the tool